rpms/microshift.yml

```diff
@@ -0,0 +1,10 @@
+content:
+  source:
+    git:
+      branch:
+        target: main
```

Comment:
this could use some explanatory text.
https://issues.redhat.com/browse/ART-4179
`main` is indeed against 4.10 right now. they do need to have test platform set up release-specific branch management before we start work on 4.12, but that would just get in the way for the current state of affairs.

Reply:
```suggestion
        target: main
        # [lmeyer 2022-07-10] main branch is indeed for 4.10 right now and that is where development is occurring.
        # We will want this to be release-4.y branches managed by Test Platform once work is beginning on
        # other releases but that would just get in the way of dev for the current state of affairs.
```